cms/models.py

```diff
@@ -72,6 +72,42 @@ def save(self, *args, **kwargs):
         super(LearningOutcomesPage, self).save(*args, **kwargs)
 
 
+class ForTeamsPage(CourseProgramChildPage):
```

Comment:
nit: Can't we simply just call it `TeamsPage` or `TeamsSubPage`?

Reply:
Named after the section it appears as on the front-end. Don't want to introduce any confusion in the future.